default.nix

```diff
@@ -1,17 +1,19 @@
 let
   sources = import nix/sources.nix;
 in
-{ pkgs ? import sources.release2105 {}
+{ pkgs ? import sources.release2111 { }
```

Comment:
I think it will be easier to advance to 21.11 in PrivateStorageio than to back this up to 21.05.  There are some build failures in our dependencies on 21.05 that aren't interesting in any way once we're up to 21.11 and will be annoying to fix.  Since ZKAPAuthorizer is supplying Tahoe-LAFS, the fact that everything is working on 21.11 seems like a pretty strong signal that upgrading PrivateStorageio to 21.11 (at least for Tahoe-LAFS/ZKAPAuthorizer) will be straightforward.  Although it is true I haven't really tried it out yet.  I could do that before committing to this upgrade to confirm that is going to be the easier path.